apiserver/facades/client/usermanager/usermanager_test.go

```diff
@@ -27,7 +27,6 @@ import (
 	usererrors "github.com/juju/juju/domain/access/errors"
```

Comment:
With these tests gone, there should be a slew of double writes in the tested code that no longer need to be there. Delete the export_test file while you're in there.

Reply:
I just had a look and I think these are already gone in the user-manager. The test code was setting up things in state but the actual checks on them have already been removed.

Reply:
There were a few other things setup which I have now removed.